### PR TITLE
Joe/video gallery

### PR DESCRIPTION
**Pull request by joebarteluce**
When the app is loaded, the gallery of cards loads. Currently, the cards represent the streams, but are not actual streams.  The gallery data is saved in a static array. The general framework for the front end is set up and ready to be built upon.

### CODE REVIEW

File: src/app/streamlisting.ts
Comment (justinsprecco) [inline src/app/streamlisting.ts:1]: Is this the metadata that a stream will hold on our platform?

Comment (Da-Scher) [reply src/app/streamlisting.ts:1]: Not necessarily all of it, but this is what Joe needs for his gallery page unless he drops some later.

**Comment by Da-Scher** [comment]
Does anybody have an opinion on the gallery component being moved to the index page? I think it would be pretty cool!

File: src/app/streamlisting.ts
Comment (justinsprecco) [reply src/app/streamlisting.ts:1]: Okay, just trying to get the database in sync with the frontend before I push it through. So we'll have the city and state associated with each stream? And what would the availableUnits, wifi, and laundry be about?

Comment (justinsprecco) [reply src/app/streamlisting.ts:1]: So I visited the tutorial you're referencing and see that is just boilerplate for a housing listing. So this isn't necessarily what the metadata will be. Don't mind me.

**Review by justinsprecco — APPROVED** [review]
Keep on truckin!

File: src/app/streamlisting.ts
Comment (Da-Scher) [reply src/app/streamlisting.ts:1]: actually, maybe this might be a good time for a quick refactor

Comment (justinsprecco) [reply src/app/streamlisting.ts:1]: Refactoring?

Comment (joebarteluce) [reply src/app/streamlisting.ts:1]: I just pushed an update to the fields to get rid of the boilerplate stuff. The fields now reflect the stream relation from the SQL database.

**Comment by joebarteluce** [comment]
> Does anybody have an opinion on the gallery component being moved to the index page? I think it would be pretty cool!

The index page loads the 'app-root' component, which in turn loads the 'app-gallery' component. So this gallery would appear when the index page is loaded. 

It is my intention that this it the app's landing page, being the first thing a user will see when the app is accessed.

File: src/app/streamlisting.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+export interface StreamListing {
+    id: number;
+    name: string;
+    city: string;
+    state: string;
+    photo: string;
+    availableUnits: number;
+    wifi: boolean;
+    laundry: boolean;
+  }
```
[the remark this answers is not in the document]
Comment (Da-Scher) [reply src/app/streamlisting.ts:11]: Changes made before my review was finished. Everything good now!

File: src/app/listing.service.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import { StreamListing } from './streamlisting';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ListingService {
+
+  constructor() { }
+
+  streamListingList: StreamListing[] = [
```
[the remark this answers is not in the document]
Comment (Da-Scher) [reply src/app/listing.service.ts:11]: Changes made already, approving.

**Review by Da-Scher — APPROVED** [review]
LGTM :+1:

**Pull request merged**